Error while checking build/O2/fullCI for 89058ee6be5412e600e8d4bbf7ec69c7d6829a37 at 2023-10-28 01:00:
```
## sw/BUILD/O2-latest/log
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
/sw/SOURCES/O2/12163-slc8_x86-64/0/Framework/AnalysisSupport/src/DataInputDirector.cxx:289:14: error: invalid use of incomplete type 'class o2::monitoring::Monitoring'
/sw/SOURCES/O2/12163-slc8_x86-64/0/Framework/AnalysisSupport/src/DataInputDirector.cxx:289:37: error: 'Metric' is not a member of 'o2::monitoring'
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/12163/89058ee6be5412e600e8d4bbf7ec69c7d6829a37/build_O2_fullCI/pretty.html).

--- a/Framework/AnalysisSupport/src/DataInputDirector.cxx
+++ b/Framework/AnalysisSupport/src/DataInputDirector.cxx
@@ -16,6 +16,8 @@
 #include "Headers/DataHeader.h"
 #include "Framework/TableTreeHelpers.h"
 #include "Monitoring/Tags.h"
+#include "Monitoring/Metric.h"
+#include "Monitoring/Monitoring.h"
 
 #include "rapidjson/document.h"
 #include "rapidjson/prettywriter.h"

--- a/Framework/AnalysisSupport/src/DataInputDirector.h
+++ b/Framework/AnalysisSupport/src/DataInputDirector.h
@@ -15,10 +15,14 @@
 
 #include "Framework/DataDescriptorMatcher.h"
 #include "Framework/DataAllocator.h"
-#include "Monitoring/Monitoring.h"
 
 #include <regex>
 #include "rapidjson/fwd.h"
+
+namespace o2::monitoring
+{
+class Monitoring;
+}
 
 namespace o2::framework
 {

